src/locale/locales/pt-BR/messages.po

```diff
@@ -1353,7 +1353,7 @@ msgstr "Fechar visualizador de imagens"
 
 #: src/components/dms/MessagesNUX.tsx:162
 msgid "Close modal"
-msgstr ""
+msgstr "Fechar modalidade"
```

Comment:
I think "modal" should be translated to "janela"

Reply:
Ok